presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -1640,6 +1640,155 @@ public void testRollupOverUnion()
     }
 
     @Test
+    public void testGrouping()
+            throws Exception
+    {
+        assertQuery("" +
```

Comment:
fix indentation so continuation of string differs with the next argument of `assertQuery`